rlclientlib/live_model_impl.cc

```diff
@@ -605,6 +624,6 @@ namespace reinforcement_learning {
       {
         complete_ids[i] = boost::uuids::to_string(boost::uuids::random_generator()()) + std::to_string(seed_shift);
       }
-    }    
+    }
```

Comment:
whitespace

Reply:
added back in. I will remove trailing white spaces on different PR